### PR TITLE
API tests

### PR DESCRIPTION
**Pull request by phi-line**
Added tests for major endpoints `/single` and `/batch`

### CODE REVIEW

**Comment by phi-line** [comment]
Fixes #16

File: server.py [hunk 1]
```diff
@@ -15,14 +15,19 @@
...
 
 COURSE_PATTERN = '[FD]0*(\d*\w?)\.?\d*([YWZH])?'
 DAYS_PATTERN = f"^{'(M|T|W|Th|F|S|U)?'*7}$"
 
 FH_TYPE_ALIAS = {'standard': None, 'online': 'W', 'hybrid': 'Y'}
 DA_TYPE_ALIAS = {'standard': None, 'online': 'Z', 'hybrid': 'Y'}
 
+try:
+    database = TinyDB(join(DB_ROOT, 'database.json'))
+except FileNotFoundError:
```
Comment (TryExceptElse) [inline server.py]: Is 'database' ever used?

File: server.py [hunk 2]
```diff
@@ -15,14 +15,19 @@
 
 DB_ROOT = 'db/'
 
-CAMPUS_LIST = ['fh', 'da']
+CAMPUS_LIST = ['fh', 'da', 'test']
```
Comment (TryExceptElse) [inline server.py]: Since testing now uses data stored in test/test_db/ , is this needed?

File: server.py (hunk 1 above)
Comment (phi-line) [reply server.py]: This line is depreciated. Thanks

File: server.py (hunk 2 above)
Comment (phi-line) [reply server.py]: I'm temporarily adding it to be of use in production when the regular databases are in flux. Like right now with second spring. Should something like this happen again, I want some static database that developers can use to test no matter what.

Comment (TryExceptElse) [reply server.py]: Are we intending to permanently maintain this test db in its current state?

**Pull request merged**